cumulusci/tasks/salesforce/Deploy.py

```diff
@@ -117,6 +143,3 @@ def _process_meta_xml(self, zipf):
         )
         zipf = zip_clean_metaxml(zipf, logger=self.logger)
         return zipf
-
-    def _write_zip_file(self, zipf, root, path):
```

Comment:
`_write_zip_file` usage is replaced in two spots in the changes Lines 76 of `_get_files_to_package` and 86 of `_get_package_zip`